packages/react-dom/src/events/ReactDOMEventListener.js

```diff
@@ -177,14 +182,17 @@ export function addTrappedEventListener(
     targetContainer = document;
   }
 
-  const validTargetContainer = ((targetContainer: any): EventTarget);
+  const targetContainerToUse =
```

Comment:
I don't think this rename is meaningfully different, but ok. Maybe just reassign on targetContainer?